pkgs/games/uqm/3dovideo.nix

```diff
@@ -13,13 +13,15 @@ let
     slides.intro = 3DOVID:addons/3dovideo/intro/intro.duk
   '' + lib.concatMapStrings makeSpin (lib.range 0 24));
 
-  helper = with haskellPackages; mkDerivation {
+  helper = with haskellPackages; mkDerivation rec {
     pname = "uqm3donix";
```

Comment:
any idea why this is not built?

Reply:
it is an optional dependency of uqm

https://github.com/NixOS/nixpkgs/commit/1aa68dd29f25b8a00e3edb70bb69a16facf2b653#diff-b737c004068cabdc5982c2507b2859559f0cca4d37cdfa686c400e716de4e393R53

it would be good to test it. have to check how

Reply:
> @aszlig how to test if uqm3donix builds without having the CD?

Unfortunately, the 3DO content is non-free, so I think there is no way to test it easily.

Just tested your branch with the following diff applied and it works:

```diff
diff --git a/pkgs/games/uqm/default.nix b/pkgs/games/uqm/default.nix
index a06c1b3b665..5b67b9bce94 100644
--- a/pkgs/games/uqm/default.nix
+++ b/pkgs/games/uqm/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchurl, pkg-config, libGLU, libGL
+{ stdenv, lib, fetchurl, fetchFromGitHub, pkg-config, libGLU, libGL
 , SDL, SDL_image, libpng, libvorbis, libogg, libmikmod
 
 , use3DOVideos ? false, requireFile ? null, writeText ? null
@@ -12,7 +12,7 @@ assert use3DOVideos -> requireFile != null && writeText != null
 
 let
   videos = import ./3dovideo.nix {
-    inherit stdenv lib requireFile writeText fetchurl haskellPackages;
+    inherit stdenv lib requireFile writeText fetchFromGitHub haskellPackages;
   };
 
   remixPacks = lib.imap1 (num: sha256: fetchurl rec {

```

Reply:
@Stunkymonkey can you apply that change too?

Reply:
should work now